source.go

```diff
@@ -175,16 +175,32 @@ type SourceObjectParams struct {
 // SourceMandateAcceptanceParams describes the set of parameters allowed for the `acceptance`
 // hash on source creation or update.
 type SourceMandateAcceptanceParams struct {
+	Date      *int64                                `form:"date"`
+	IP        *string                               `form:"ip"`
+	Offline   *SourceMandateAcceptanceOfflineParams `form:"offline"`
+	Online    *SourceMandateAcceptanceOnlineParams  `form:"online"`
+	Status    *string                               `form:"status"`
+	Type      *string                               `form:"type"`
+	UserAgent *string                               `form:"user_agent"`
+}
+
+type SourceMandateAcceptanceOnlineParams struct {
 	Date      *int64  `form:"date"`
 	IP        *string `form:"ip"`
-	Status    *string `form:"status"`
```

Comment:
Why is this disappearing? It's a breaking change to remove this.

Reply:
@remi-stripe  I think it is still there, see line 182 on the right.
It is just that Github diff is not smart enough :)